### DISCUSSION TITLE
Why my github account blocked by Vercel?

### DISCUSSION THREAD

**Discussion by wormtql** [vercel/community#494] [Help] [answered]
I tried to login Vercel with github, but it told me `This user account is blocked. Contact registration@vercel.com for more information.`
I believe I did nothing wrong, in my memory it's even my first time using Vercel. I contacted Vercel support once which did not respond.

**Comment by Pieparker** [comment, marked as answer]
Hey there, our team would have replied to you, if you contacted them from registration@vercel.com. Please double-check your spam mail box!

**Comment by Siyabonga-Mthembu** [comment]
I am trying to log in with gitHub and i recieve this feedback "This user account is blocked.". what must i do to get it write?
